Tools/TestWebKitAPI/Tests/WebKitGLib/TestMultiprocess.cpp

```diff
@@ -252,8 +250,12 @@ static void testMultiprocessWebViewCreateReadyClose(UIClientMultiprocessTest* te
 
 void beforeAll()
 {
+#if !ENABLE(2022_GLIB_API)
     // Check that default setting is the one stated in the documentation
+    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
     g_assert_cmpuint(webkit_web_context_get_process_model(webkit_web_context_get_default()), ==, WEBKIT_PROCESS_MODEL_MULTIPLE_SECONDARY_PROCESSES);
+    ALLOW_DEPRECATED_DECLARATIONS_END
+#endif
```

Comment:
I don't think this is worth it, let's just remove it.

Reply:
Done.